client/dashboard/utils.js

```diff
@@ -13,3 +13,106 @@ export function getCountryCode( countryState ) {
 
 	return countryState.split( ':' )[ 0 ];
 }
+
+/**
+ * Returns currency data by country/region. Contains code, position, thousands separator, decimal separator, and precision.
+ *
+ * @format
+ * @return {object} Curreny data.
+ */
+export function getCurrencyData() {
```

Comment:
I wonder if we should move this to the `currency` package.  May be useful there too.